base_export_manager/views/res_groups.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="res_groups_view_form" model="ir.ui.view">
```

Comment:
You should use original_id here as per the guideline.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views